container/docker/factory.go

```diff
@@ -62,6 +63,10 @@ func UseSystemd() bool {
 	return useSystemd
 }
 
+func RootDir() string {
+	return *dockerRootDir
```

Comment:
not blocking this PR, but we need to detect this root dir better if we're going to start relying on it. Today we assume it is always /var/lib/docker, but users do change it.